pkgs/development/python-modules/opentelemetry-api/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, deprecated
+, hatchling
+, importlib-metadata
+, opentelemetry-test-utils
+, setuptools
+, pytestCheckHook
+, pythonRelaxDepsHook
+}:
+
+buildPythonPackage rec {
+  pname = "opentelemetry-api";
+  version = "1.17.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub
+    {
+      owner = "open-telemetry";
+      repo = "opentelemetry-python";
+      rev = "refs/tags/v${version}";
+      sha256 = "sha256-IuomrzaXCUAjTArjL+acXIzZ8VgSh0KsVTyiBaXbcdA=";
+      sparseCheckout = [ "/${pname}" ];
+    } + "/${pname}";
```

Comment:
```suggestion
  src = fetchFromGitHub {
    owner = "open-telemetry";
    repo = "opentelemetry-python";
    rev = "refs/tags/v${version}";
    hash = "sha256-IuomrzaXCUAjTArjL+acXIzZ8VgSh0KsVTyiBaXbcdA=";
    sparseCheckout = [ "/${pname}" ];
  } + "/${pname}";
```

Reply:
Thanks. I formatted all these with `nixpkgs-fmt`. But I will change to match your suggestion.